src/js/core/UpdateStream.js

```diff
@@ -0,0 +1,100 @@
+import { Observable } from "rxjs/Observable";
+import UserSettingsStore from "#SRC/js/stores/UserSettingsStore";
+import { request } from "@dcos/http-service";
+import Util from "#SRC/js/utils/Util";
+import { SAVED_STATE_KEY } from "#SRC/js/constants/UserSettings";
+import compareVersions from "compare-versions";
```

Comment:
nitpick please put the `package.json` imports first and `#SRC` imports afterwards.